services/site-scout/launch-worker.sh

```diff
@@ -91,4 +110,27 @@ done
 mkdir -p /tmp/site-scout/local-results
 
 update-ec2-status "Setup: launching site-scout"
-python3 -m site_scout ./build/firefox -i ./active_lists/ --status-report status.txt --time-limit "$TIME_LIMIT" --memory-limit "$MEM_LIMIT" --jobs "$JOBS" --url-limit "$URL_LIMIT" --fuzzmanager -o /tmp/site-scout/local-results
+python3 -m site_scout "$TARGET_BIN" -i ./active_lists/ --status-report status.txt --time-limit "$TIME_LIMIT" --memory-limit "$MEM_LIMIT" --jobs "$JOBS" --url-limit "$URL_LIMIT" --fuzzmanager -o /tmp/site-scout/local-results "$COVERAGE_FLAG"
```

Comment:
`COVERAGE_FLAG` shouldn't be quoted or you'll end up passing `""` to argparse which will confuse it I think. If it's unquoted then empty string will not create an arg.